tutorial.md

```diff
@@ -645,15 +689,48 @@ Let's copy the implementation over into our main function:
   main = xmonad
        . ewmhFullscreen
        . ewmh
-     =<< statusBarProp "xmobar" def toggleStrutsKey myConfig
+       $ withEasySB (statusBarProp "xmobar" (pure def))
+                    defToggleStrutsKey
+                    myConfig
```

Comment:
This is fine, though I would prefer having the all the status bar stuff as one modifier. This should give the users the impression that it's just another combinator that you could add and remove, so something like this:
```haskell
  main = xmonad
       . ewmhFullscreen
       . ewmh
       . withEasySB (statusBarProp "xmobar" (pure def)) defToggleStrutsKey
       $ myConfig
```
I think this puts more emphasis on the composability of the different modifiers, and just looks a bit nicer. But this is just a very minor nitpick

Reply:
Oh that's brilliant, thanks!